docs/introduction.rst

```diff
@@ -1,73 +1,51 @@
-- Low latency, high throughput data acquisition platform.
+User Interface
+==============
+The user interface is comprised of two main sections: a central section called the workspace, and the surrounding area for utility panels.
 
-Data Items and Display Items
-----------------------------
-|AppName| operates on data arrays, which include 2D images and 1D spectra, which are generally called *Data Items*.
+.. image:: graphics/workspace.png
+  :width: 320
+  :alt: Example of the Swift Workspace 
 
-Data items represent data arrays with anywhere from one to five dimensions. The data array associated with a data item can be 1D or 2D data and then organized into 1D or 2D collections or 1D sequences.
+The central workspace is for display panels which can either display a single data item or thumbnail images of multiple data items in one of the browser formats. For more information, see :ref:`display-panels`.
 
-A spectrum image would be a 2D collection of 1D data. This would be typical data acquired from a STEM microscope spectrum imaging acquisition.
+The utility panels surrounding the central workspace allow you to make adjustments to a selected data item in a display panel. For more information on their specific functions, see :ref:`user-interface`.
 
-Data Items are displayed using one or more *Display Items*.
+Data and Display
+================
+Data is stored in a project as data items. Data items can be dragged into display panels to be displayed. Data items that are displayed in a display panel can be annotated with graphics, processed with computations, and more. These displayed data items are referred to as display items. 
 
-See :ref:`data-items`.
+Read more in :ref:`data-items`.
 
-Projects
---------
-Data items are organized into projects. Projects are stored on disk and take the form of an index file and an associated data directory.
+Graphics and Masks
+==================
+Use graphics to highlight portions of the data, select areas for processing, or focus on specific sections of data.
 
-You can have any number of projects, but only a single one can be open at once. When you open a project, it is associated with a main document window.
+Masking allows you to block out part of an image and manipulate only a specific section of the data. Use a preset mask or add graphics to a mask to precisely define your area of interest.
 
-See :ref:`data-management`.
+See :ref:`graphics` for more information.
 
-Workspaces
-----------
-The main window is organized around a main area called a workspace. A workspace has a layout which includes one or more display panels in which display items can be visualized.
+Processing and Analysis
+=======================
+Processing data by adding line profiles to an image, running Python scripts, filtering data, etc. can give you deeper insight into your data. Use the :ref:`inspector panel` to adjust processing settings. Customizing processing allows for easier and more specific analysis of data.
```

Comment:
Similar to other comments I've made, I think this is too "sales-y". I'd rather this just be an explanation followed by a few examples. I'm not convinced my explanation is notably better, but I think it provides a better starting point:

"Nion Swift can automatically perform processing and analysis that is performed when data or parameters change as the result of acquisition, user-initiated parameter changes, or regions of interest edits."

"Use the inspector panel..."

I would leave out the mention of the line profile since it is very specific and not easy to explain in this section.

You could include an example to the effect of "you can do processing on an image by making a rectangle to serve as the region of interest, then choosing Processing > Filter > Sobel..." but that is probably too much detail for this section.